src/pages/webgl/03_zippedFile.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useRef } from 'react'
+import { useWebgl } from './webglPageUtils'
+
+export function WebglZippedFile () {
+  const div = useRef<HTMLDivElement>(null)
+
+  useWebgl(div, (webgl) => {
+    webgl.loader.request({ url:'https://vim.azureedge.net/samples/residence.vim'})
```

Comment:
We should be moving onto storage.cdn.vimaec.com as our CDN URL since vim.azureedge.net is going away
```suggestion
    webgl.loader.request({ url:'https://storage.cdn.vimaec.com/samples/residence.vim'})
```